src/main/java/com/pontusvision/graphutils/globals.groovy

```diff
@@ -851,8 +859,13 @@ class PontusJ2ReportingFunctions {
     return System.getenv(envVar)
   }
 
-  static String formatDateNow(String pattern) {
-    DateTimeFormatter formatter = DateTimeFormatter.ofPattern(pattern)
+
```

Comment:
Please create a new function with these new parameters; otherwise, we break backwards compatibility